include/z64environment.h

```diff
@@ -74,6 +74,27 @@ typedef enum WeatherMode {
     /* 5 */ WEATHER_MODE_HEAVY_RAIN // scene must define settings for light config 4
 } WeatherMode;
 
+typedef enum {
+// Clear Sky
```

Comment:
```suggestion
    // Clear Sky
```
same for below comments